pkg/util/cluster/cluster.go

```diff
@@ -501,30 +511,68 @@ func (c *Cluster) registerSubscription(ctx context.Context) error {
 		return err
 	}
 
-	req, err := http.NewRequest(http.MethodPut, "https://localhost:8443/subscriptions/"+c.env.SubscriptionID()+"?api-version=2.0", bytes.NewReader(b))
+	req, err := http.NewRequest(http.MethodPut, localDefaultURL+"/subscriptions/"+c.env.SubscriptionID()+"?api-version=2.0", bytes.NewReader(b))
 	if err != nil {
 		return err
 	}
 
 	req.Header.Set("Content-Type", "application/json")
 
-	cli := &http.Client{
-		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{
-				InsecureSkipVerify: true,
-			},
-		},
-	}
-
-	resp, err := cli.Do(req)
+	resp, err := insecureLocalClient.Do(req)
 	if err != nil {
 		return err
 	}
 
 	return resp.Body.Close()
 }
 
-func (c *Cluster) insertDefaultVersionIntoCosmosdb(ctx context.Context) error {
+// getVersionsInCosmosDB connects to the local RP endpoint and queries the
+// available OpenShiftVersions
```

Comment:
Pure curiosity:

I'm still new to go and one of the things I find very odd is the lack of recommended line lengths. Here you seem to have tightened your comments to ~80 but longer in the code. Is there a reason for this/some convention I'm unaware of?

Reply:
No, i don't think there's a convention regarding that. I wrap a line when it feels like it's getting too long, and for godoc comments i just like the look of them them being wrapped :)